### PR TITLE
ft-raystarhome.rw

### PR DESCRIPTION
**Pull request by emmymbaraga**
editing template to raystarhome.rw website

### CODE REVIEW

**Review by fridolinho — CHANGES_REQUESTED** [review]
1. rename service menu to what we do and remove collections menu
2. Rename About menu to who we are
3. pull down who we are section and merge it with know more about us section
4. Remove existing what we section.
5. Rename collection section to what we do with just 2 categories(our collections and House for rent)
6. Remove social media icons.

**Comment by fridolinho** [comment]
[raystarhome.rw](https://trello.com/c/yWQMVOQi/150-raystarhomerw)

File: raystarhome.rw/components/gallery.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<div class="container" id="product">
+   
+ <div class="wrapper">
+     <h2 class="text-center header-size font-size">Our Collections</h2>
+     <nav>
+        <div class="items">
```
Comment (Ramadhan0) [inline raystarhome.rw/components/gallery.html:6]: ###  These buttons are too big for smaller screens.  
sss

- make them responsive and youre good to go.

**Review by Ramadhan0 — CHANGES_REQUESTED** [review]
Fix the responsiveness @emmymbaraga

**Review by fridolinho — APPROVED** [review]

**Pull request closed without merging**